docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -56,6 +73,8 @@ const FILTER_CHIPS = [
 
 const componentNameToKebabCase = (componentName) => componentName.replaceAll(' ', '-').toLowerCase()
 
+const componentPath = (component) => component.path ? `/vue-components/${component.path}` : `/vue-components/${componentNameToKebabCase(component.name)}`
```

Comment:
I'd use destructuring here, since you don't need the whole object reference anymore
And move the ternary inside the template string to avoid repeating `vue-components`